### PR TITLE
Add gatsby-plugin-image

### PR DESCRIPTION
**Pull request by Razzeee**
This will hopefully help with the images.

I think I migrated all static images and we need to use https://www.gatsbyjs.com/plugins/gatsby-plugin-image#dynamic-images for all other `` tags. But I'm not sure how to do that, as it seems it will need the image paths to be send via graphql?

### CODE REVIEW

File: src/components/Social.tsx [hunk 1]
```diff
@@ -72,9 +72,9 @@ function Social() {
             viewBox="0 0 24 24"
             fill="none"
             stroke="currentColor"
-            strokeWidth="2"
```
Comment (pkscout) [inline src/components/Social.tsx:75]: React DOM names got overwritten with the HTML ones for some reason.

**Review by pkscout — CHANGES_REQUESTED** [review]
see comments

**Comment by pkscout** [comment]
I don't know what you did differently than I have, but sharp finally built at Netlify using this PR. So once the Social icon DOM stuff is fixed I can merge it. When you say you've migrated all the static images, does that include the addon images? Those are downloaded dynamically from the mirrors when add-ons are added/updated, so I just want to see if there's something more we need to add to the addonparser script to deal with this.

File: src/components/SectionWidgets.tsx [hunk 2]
```diff
@@ -79,15 +80,19 @@ function AboutOfficialRemotes() {
       <div className="flex justify-start">
         <div>
           <span>Kore™ (Android)</span>
-          <img
+          <StaticImage
             className="w-200 h-200"
             alt="Kore QR Code"
-            src="/images/kore-qr.webp"
```
Comment (pkscout) [inline src/components/SectionWidgets.tsx:85]: I don't understand why, but somehow this change causes the remote name to show up below the image and not lined up.
Screen Shot 2021-03-28 at 7 04 25 AM

File: src/components/Social.tsx (hunk 1 above)
Comment (Razzeee) [reply src/components/Social.tsx:75]: I changed them manually to the `-` ones, as vscode complains. And we had a mixed style in that file (I'm also getting build errors for these)

File: src/components/SectionWidgets.tsx (hunk 2 above)
Comment (Razzeee) [reply src/components/SectionWidgets.tsx:85]: Fixed

**Pull request merged**